editoast/src/models/study.rs

```diff
@@ -70,15 +68,17 @@ pub struct Study {
     #[diesel(deserialize_as = i32)]
     #[derivative(Default(value = "Some(0)"))]
     pub budget: Option<i32>,
-    #[diesel(deserialize_as = Vec<String>)]
+    #[diesel(deserialize_as = Vec<Option<String>>)]
     #[derivative(Default(value = "Some(Vec::new())"))]
-    pub tags: Option<Vec<String>>,
+    pub tags: Option<Vec<Option<String>>>,
```

Comment:
I still don't get it

Reply:
We decided to use `#[diesel(deserialize_as = "…")]`, as mentionned in the [diesel migration notes](https://diesel.rs/guides/migration_guide.html#2-0-0-nullability-of-array-elements)